playbooks/common/openshift-master/scaleup.yml

```diff
@@ -0,0 +1,42 @@
+---
+- include: ../openshift-cluster/evaluate_groups.yml
+
+- name: Gather facts
+  hosts: oo_etcd_to_config:oo_masters_to_config:oo_nodes_to_config
+  roles:
+  - openshift_facts
+
+- name: Update master count
+  hosts: oo_masters:!oo_masters_to_config
+  serial: 1
+  vars:
+    openshift_master_count: "{{ groups.oo_masters | length }}"
+  roles:
+  - openshift_facts
+  post_tasks:
+  - name: Update master count
+    lineinfile:
+      dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
+      regexp: '^  masterCount:.*$'
+      line: "  masterCount: {{ openshift_master_count }}"
```

Comment:
Any objection to having a generic-ish Ansible module for modifying a yaml file that would parse the yaml file modify a specify key in jinja2 '.' notation and then write the file back out?

I'm a bit worried that we may have unintentional breakage if the config file is not in a pretty printed format.